### DISCUSSION TITLE
Linux Releases

### DISCUSSION THREAD

**Discussion by TonyStone31** [zamtmn/zcad#60] [Ideas]
Linux releases would be very welcomed.  Have you ran zcad on linux at all?  Is there some windows dependency?   I have tried building it on my machine but ran into too many issues.  I will try to build it again some day when I have more free time.

**Comment by TonyStone31** [comment]
Ahh just saw you had a Linux release in 2018.  If you can do a newer Linux build that would be awesome!  I will try again tonight myself I guess but I don't have a lot of time.  :)

**Comment by zamtmn** [comment]
Hi!
There are no dependencies on Windows. In Linux code is compile and working, I've had been no testing in Linux lately, so of course there will be bugs. GTK2 version should be more workable than qt. I will make the next release for linux. I will also accept help in writing a script to create deb\rpm packages, and in general any help))

**Comment by zamtmn** [comment]
https://github.com/zamtmn/zcad/blob/master/BUILD_FROM_SOURCES.md
detailed assembly instructions. it is focused on Windows, but everything is exactly the same in linux, only simpler))
